extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -146,26 +134,24 @@ class AmpStickyAd extends AMP.BaseElement {
    * @private
    */
   addCloseButton_() {
-    if (this.closeButton_ && this.boundOnClick_) {
+    if (this.closeButton_) {
       return;
     }
     const closeButton = this.getWin().document.createElement('div');
-    closeButton.setAttribute('class', '-amp-sticky-ad-close-button');
-    closeButton./*OK*/innerHTML = '&#10006';
+    closeButton.classList.add('-amp-sticky-ad-close-button');
+    closeButton.textContent = '&#10006';
```

Comment:
This will need an aria label. If there is nothing semantic, we might need to require publishers to provide the label.